docs/source/app-dev/services.rst

```diff
@@ -157,9 +157,9 @@ You can get these included in requests related to Transactions by setting the ``
 Active contracts service
 ========================
 
-Use the **active contracts service** to obtain a party-specific view of all contracts currently active on the ledger.
+Use the **active contracts service** to obtain a party-specific view of all contracts that are active on the ledger at the time of the request.
 
-The active contracts service returns the current contract set as a set of created events that would re-create the state being reported. Each created event has a ledger offset where it occurs. You can infer the ledger offset of the contract set from the ledger offset of the last event you receive.
+The active contracts service returns its response as a stream of batches of the created events that would re-create the state being reported (the size of these batches is left to the ledger implementation). After all contracts have been reported, one final message includes the offset at which this was the state. This offset can be used to subscribe to the "flat transactions" stream to keep a consistent view of the active contract set without querying the active contract service further.
```

Comment:
The description "after all contracts have been reported" makes it sound like we guarantee that the message with the offset has no contracts itself. Is that actually a guarantee we want to make?

Reply:
Addressed by https://github.com/digital-asset/daml/pull/10880/commits/f98677d5f45b590dabe3da3948be382d79a810e1